### PR TITLE
Basic GitHub enricher

### PR DESCRIPTION
**Pull request by jsmrcka**

### CODE REVIEW

**Review by Sgitario — APPROVED** [review]
lgtm!

File: enrichers/src/main/java/io/quarkus/qe/Dispatcher.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class Dispatcher {
+
+    @Inject
+    GitHubEnricher gitHubEnricher;
```
Comment (Sgitario) [inline enrichers/src/main/java/io/quarkus/qe/Dispatcher.java:16]: In the future, this should be a list of populators.

**Review by Sgitario — CHANGES_REQUESTED** [review]
Can you configure the enricher to use the HTTP PORT 8083 and add the smallrye health check?

**Review by Sgitario — APPROVED** [review]

File: enrichers/src/main/java/io/quarkus/qe/Dispatcher.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    @Incoming(Channels.ENRICH_REPOSITORY)
```
Comment (pjgg) [inline enrichers/src/main/java/io/quarkus/qe/Dispatcher.java:18]: missing acknowledgment strategy

**Review by pjgg — CHANGES_REQUESTED** [review]

**Pull request merged**